packages/server/lib/modes/run.ts

```diff
@@ -310,20 +310,28 @@ async function startVideoRecording (options: { previous?: VideoRecording, projec
   return videoRecording
 }
 
-const warnVideoRecordingFailed = (err) => {
-  // log that post processing was attempted
+const warnVideoCaptureFailed = (err) => {
+  // log that capturing video was attempted
   // but failed and don't let this change the run exit code
-  errors.warning('VIDEO_POST_PROCESSING_FAILED', err)
+  errors.warning('VIDEO_CAPTURE_FAILED', err)
 }
 
-async function postProcessRecording (options: { quiet: boolean, videoCompression: number | boolean, shouldUploadVideo: boolean, processOptions: Omit<ProcessOptions, 'videoCompression'> }) {
+const warnVideoCompressionFailed = (err) => {
+  // log that compression was attempted
+  // but failed and don't let this change the run exit code
+  errors.warning('VIDEO_COMPRESSION_FAILED', err)
+}
+
+async function compressRecording (options: { quiet: boolean, videoCompression: number | boolean, shouldUploadVideo: boolean, processOptions: Omit<ProcessOptions, 'videoCompression'> }) {
   debug('ending the video recording %o', options)
 
   // once this ended promises resolves
-  // then begin processing the file
-  // don't process anything if videoCompress is off
+  // then begin compressing the file
+  // don't compress anything if videoCompress is off
   // or we've been told not to upload the video
   if (options.videoCompression === false || options.videoCompression === 0 || options.shouldUploadVideo === false) {
+    debug('skipping compression')
```

Comment:
added a debug log if this condition is met since otherwise its hard to tell. I wonder if we want to add any of these options to the telemetry span in the future to determine if this is a no-op (we can do it in this PR as well just wanted to see if others thought it was a good idea).

Reply:
videoname, videoCompression and compressedVideoName are added to the string now. You could add 'should upload' if you want. (down on line 685)

Reply:
I missed that. I think that should be enough for us to infer the behavior!